website/go.mod

```diff
@@ -1,7 +1,5 @@
 module github.com/cloudoperators/greenhouse/website
 
-go 1.20
+go 1.24.2
```

Comment:
It should be fine to leave `1.24.0` as the minimum go version. This provides usually provides less friction with local/ci environments.
```suggestion
go 1.24.0
```

Reply:
It is populated from the templater now when installing it.